templates/infrastructure/locals.tf

```diff
@@ -4,7 +4,7 @@ locals {
   #Enable this to provision VPC cluster.
   create_cluster = true
   #True to create new Observability Services. False if Observability Service instances are already existing.
-  enable_observability = true
+  enable_observability = false
```

Comment:
create the same `enable_observability` variable in the variables file and use it instead of this local var in main.if 
(use var.enable_observability instead of local.enable_observability)